tutorials/vpc-private-gateway-autoconfiguration/index.mdx

```diff
@@ -0,0 +1,202 @@
+
+6. Select the Private Network from the drop-down list, enable the DHCP server and dynamic NAT. Then click **Attach Private Network**:
+
+  <Lightbox src="scaleway_vpc_public_gateway_attach_network.png" alt="" /> 
+
+7. Check if the IP address for the Private Network has been assigned to the instance using the DHCP server of the Private Gateway by running the `ip a` command: 
+
+  ```
+  3: ens5: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
+    link/ether 02:00:00:00:5a:d8 brd ff:ff:ff:ff:ff:ff
+    inet 192.168.0.2/24 brd 192.168.0.255 scope global dynamic ens5
+       valid_lft 3513sec preferred_lft 3513sec
+    inet6 fe80::ff:fe00:5bea/64 scope link 
+       valid_lft forever preferred_lft forever
+  ```
+
+  You can see the private IP address `192.168.0.2` has been configured automatically on the interface.
+
+## Adding a second Instance to the Private Gateway
+
+1. Click **Instances** in the **Compute** section of the Scaleway console to display the list of your instances. Click **+ Create an Instance**.
+
+2. Enter the details for your new instance and click **Create a new Instance**. The instace is being created. Once this is done, the list of your Instances displays.
+
+3. Click **VPC** in the **Network** section. The list of your Private Networks displays.
+
+4. Click <icon name="more" /> > **More info** to display the details of the Private Network.
+
+5. Click the **Attached Ressources** tab to display a list of resources attached to your Private Network.
+
+7. Select your new Instance in the drop down list and click **Add Instance** to attach it to the Private Network:
+
+  <Lightbox src="scaleway_vpc_add_instance.png" alt="" />
+
+  The new Instance displays in the list of devices connected to the Private Network.
+
+## Configuring port forwarding 
+
+Port forwarding allows to configure a mapping of ports on the IP address of the Public Gateway to IP addresses and ports inside the Private Network. This allows to access services on Virtual Instances in a Private Network without need for a flexible public IP on them. 
+
+1. Click **VPC** in the **Network** section of the side menu in the Scaleway console. The VPC configuration displays.
+
+2. Click the **Public Gateways** tab. The list of your Public Gateways displays. 
+
+3. Click <Icon name="more" /> > **More Info** to display the Public Gateway information. 
+
+4. Click the **DHCP** tab. Then click **View DHCP Status** to display the list of devices autoconfigured by the DHCP server: 
+
+  <Lightbox src="scaleway_vpc_dhcp_status.png" />
+
+  <Message type="tip">
+
+  You notice in the screenshot above, that both entries are **dynamic** types. This means the DHCP server dynamically assigns an IP address to each device. The assigned IP address may change and the port forwarding will stop working. To avoid this, configure a **static** lease. 
```

Comment:
```suggestion
  As you can notice in the screenshot above, both entries have the **dynamic** type. This means the DHCP server has dynamically assigned an IP address to each device. The assigned IP address may change and the port forwarding will stop working. To avoid this, configure a **static** lease. 
```